### DISCUSSION TITLE
Human does not work on Motorola e5 plus

### DISCUSSION THREAD

**Discussion by pooldeveloper** [vladmandic/human#218] [Q&A]
Human does not work on Motorola e5 plus

![image](https://user-images.githubusercontent.com/74687129/142089088-115dd9af-9539-47df-869c-59b646860577.png)

It remains in initialization and does not turn on the camera or ask for permission, nothing

What can be the solution, try to change the backend in the settings but it doesn't work or the "human" doesn't have support for Motorola?

Thanks in advance

**Comment by vladmandic** [comment]
which browser? which version of android? provide full information when posting an issue. and if its an issue, post an issue, not a discussion topic.

also note that phone in question is very much underpowered, even if human worked it will be really slow. at the very least, it wont be able to run with default models enabled in a device with just 2gb of ram total.

**Comment by pooldeveloper** [comment]
Hi @Vladmandic sorry for the delay

*** Moto E5 Plus ***
   -Operating system: Android Oreo 8.0
   -RAM memory: 2RAM
   -Screen: 5.99 "
   -Processor: Snapdragon 435
   -Front camera: 5 MP
   -Browser: Google Chrome

In case you don't support "Human" in these devices, could you see a solution for these?

**Comment by vladmandic** [comment]
not really. you can try different libraries, but simple fact is that even if something does work, it will be really slow.

**Reply by pooldeveloper** [reply]
ok thanks for your time @vladmandic.
Would you have a series of specifications of when "Human" works with mobile devices or only if it has more than 2RAM would it work well?

**Reply by vladmandic** [reply]
Chances are if it has 4GB, it has a better GPU as well, so that's good start (E5 has a really bad GPU).

**Reply by pooldeveloper** [reply]
ok thanks @vladmandic
